packages/kolibri-common/components/SyncSchedule/SyncSchedule.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+
+  <EditSyncSchedule msg="Remove device from sync schedule" />
```

Comment:
I don't think this needs to be a separate component - and this message needs to be made translatable. I would recommend moving this message into the `EditSyncSchedule` component, and conditionally displaying it if there is an existing scheduled sync.